.github/workflows/deploy_to_npm.yaml

```diff
@@ -1,10 +1,11 @@
-name: Publish Packages
+name: Build & Deploy
 
 on:
     workflow_dispatch:
     push:
         branches:
             - release
+            - master
```

Comment:
this will also release on master branch?

Reply:
yes, but there is a filter in the deployment step, so it does not publish npms, bu only the documentation and examples, it matches the old workflow